### PR TITLE
Finish mavenization, added analysis features

### PR DESCRIPTION
**Pull request by FabiScheytt**

### CODE REVIEW

File: bundles/de.fzi.bwcps.stream.analysis/src/de/fzi/bwcps/stream/analysis/BWCPSAnalyzer.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import org.eclipse.emf.common.notify.Notifier;
+import org.eclipse.emf.ecore.EObject;
+import org.eclipse.emf.ecore.util.EcoreUtil;
+
+import de.fzi.bwcps.stream.bwcps_streams.commons.NamedElement;
+import de.fzi.bwcps.stream.bwcps_streams.entity.StreamRepository;
+
+public class BWCPSAnalyzer {
+	public void start(NamedElement startingPoint) {
+		System.out.println(startingPoint);
```
Comment (joehen) [inline bundles/de.fzi.bwcps.stream.analysis/src/de/fzi/bwcps/stream/analysis/BWCPSAnalyzer.java]: Does it make sense to use a logger here?

File: bundles/de.fzi.bwcps.stream.analysis/src/de/fzi/bwcps/stream/analysis/BWCPSAnalyzer.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+		Notifier rootObj = EcoreUtil.getRootContainer(startingPoint, true);
+		if (rootObj instanceof StreamRepository) {
+			EObject root = (EObject) rootObj;
```
Comment (joehen) [inline bundles/de.fzi.bwcps.stream.analysis/src/de/fzi/bwcps/stream/analysis/BWCPSAnalyzer.java]: cast to StreamRepository or change if statement

File: bundles/de.fzi.bwcps.stream.analysis/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: BWCPS Stream Model Analysis Tools
+Bundle-SymbolicName: de.fzi.bwcps.stream.analysis;singleton:=true
+Bundle-Version: 0.1.0.qualifier
+Bundle-Vendor: FZI Forschungszentrum Informatik
+Automatic-Module-Name: de.fzi.bwcps.stream.analysis
+Bundle-RequiredExecutionEnvironment: JavaSE-13
```
Comment (joehen) [inline bundles/de.fzi.bwcps.stream.analysis/META-INF/MANIFEST.MF]: please stay with Java 11

File: bundles/de.fzi.bwcps.stream.analysis/.classpath [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-13"/>
```
Comment (joehen) [inline bundles/de.fzi.bwcps.stream.analysis/.classpath]: java 11 where possible as minimum requirement

File: bundles/de.fzi.bwcps.stream.design/META-INF/MANIFEST.MF [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+Bundle-Version: 1.0.0.qualifier
+Bundle-Activator: de.fzi.bwcps.stream.design.Activator
+Bundle-Localization: plugin
+Require-Bundle: org.eclipse.ui,
+ org.eclipse.core.runtime,
+ org.eclipse.core.resources,
+ org.eclipse.sirius,
+ org.eclipse.sirius.common.acceleo.aql
+Bundle-ActivationPolicy: lazy
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
```
Comment (joehen) [inline bundles/de.fzi.bwcps.stream.design/META-INF/MANIFEST.MF]: Java 11

**Review by joehen — CHANGES_REQUESTED** [review]

File: bundles/de.fzi.bwcps.stream.analysis/src/de/fzi/bwcps/stream/analysis/BWCPSAnalyzer.java (hunk 1 above)
Comment (FabiScheytt) [reply bundles/de.fzi.bwcps.stream.analysis/src/de/fzi/bwcps/stream/analysis/BWCPSAnalyzer.java]: I think you caught an old commit for your review.
A logger makes sense anyway.

File: bundles/de.fzi.bwcps.stream.analysis/src/de/fzi/bwcps/stream/analysis/BWCPSAnalyzer.java (hunk 2 above)
Comment (FabiScheytt) [reply bundles/de.fzi.bwcps.stream.analysis/src/de/fzi/bwcps/stream/analysis/BWCPSAnalyzer.java]: Again, this changed since the first commit ...

**Pull request merged**